confluence-xml/src/test/resources/confluencexml/include.test

```diff
@@ -0,0 +1,96 @@
+.#------------------------------------------------------------------------------
+.expect|filter+xml
+.# Various links and reference related tests
```

Comment:
Wrong comment ?

Would be interesting to have in there at least an example with space information in the parameter and another example which is impacted by the entity name validator (see "*tovalidate" examples in `links` tests).